source/write/run-command.txt

```diff
@@ -0,0 +1,180 @@
+.. _php-run-command:
+
+======================
+Run a Database Command
+======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: administration, code example
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+php-library+}
+to run a database command. You can use database commands to perform a
+variety of administrative and diagnostic tasks, such as fetching server
+statistics, initializing a replica set, or running an aggregation pipeline.
+
+.. important:: Prefer Library Methods to Database Commands
+
+   The library provides wrapper methods for many database commands.
+   We recommend using these methods instead of executing database
+   commands when possible.
+   
+   To perform administrative tasks, use the :mongosh:`MongoDB Shell </>`
+   instead of the {+php-library+}. Calling the ``db.runCommand()``
+   method inside the shell is the preferred way to issue database
+   commands, as it provides a consistent interface between the shell and
+   drivers or libraries.
```

Comment:
I don't follow the last sentence here. How does `db.runCommand()` provide a "consistent interface between the shell and drivers or libraries"?

I would probably call out that the shell provides many more wrapper methods (aka helpers) for various administration commands: https://www.mongodb.com/docs/manual/reference/method/

And _those_ should probably also be preferred to running `db.runCommand()` or a driver's [command helper](https://github.com/mongodb/specifications/blob/master/source/run-command/run-command.md).

The selling point of the command helper is that it's a primitive tool to run any command for which the shell or driver doesn't already provide a wrapper/helper.

Reply:
Reworded the language here to make it clear that there's a hierarchy of preferred ways to perform tasks